mkdocs.yml

```diff
@@ -84,12 +84,14 @@ extra:
   social:
     - icon: fontawesome/brands/github
       link: https://github.com/bids-standard/bids-specification/
-    - icon: fontawesome/brands/twitter
-      link: https://twitter.com/BIDSstandard/
+    - icon: fontawesome/brands/x-twitter
+      link: https://x.com/BIDSstandard/
```

Comment:
Didn't someone make a BlueSky? Or did we just talk about it?

Reply:
Whoops I missed it above.

Reply:
yes, we could add bluesky and youtube to the list as well :-)

Reply:
No no I just didn't read through the PR carefully before commenting.

Reply:
However, a bluesky icon seems to be missing as of yet:

- https://github.com/FortAwesome/Font-Awesome/issues/19810